tests/sentry/rules/processing/test_delayed_processing.py

```diff
@@ -1326,6 +1332,66 @@ def test_apply_delayed_process_count_then_percent(self, safe_execute_callthrough
         self._assert_count_percent_results(safe_execute_callthrough)
 
 
+class ProcessRuleGroupsInBatchesTest(CreateEventTestCase):
+    def setUp(self):
+        super().setUp()
+
+        self.project = self.create_project()
+        self.group = self.create_group(self.project)
+        self.group_two = self.create_group(self.project)
+        self.group_three = self.create_group(self.project)
+        self.rule = self.create_alert_rule()
+
+    @patch("sentry.rules.processing.delayed_processing.apply_delayed")
+    def test_no_redis_data(self, mock_apply_delayed):
+        mock_delayed = Mock()
+        mock_apply_delayed.delayed = mock_delayed
+        process_rulegroups_in_batches(self.project.id)
+
+        mock_delayed.assert_called_once_with(self.project.id)
```

Comment:
nit: I think any calls on mocks generate more mocks, so you can do:
```suggestion
        process_rulegroups_in_batches(self.project.id)

        mock_apply_delayed.delayed.assert_called_once_with(self.project.id)
```